frontend/templates/views/partials/search.j2

```diff
@@ -76,63 +76,61 @@
                     reset-on-refresh
                     single-item
           >
-            {% raw %}
             <template type="amp-mustache">
               <div class="ap-o-search-result-list">
-                {{^prevUrl}}
-                {{#result.components.length}}
-                <h4 class="ap-o-search-result-category">{% endraw %}{{_('Components')}}{% raw %}</h4>
-                {{/result.components.length}}
-                {{/prevUrl}}
-                {{#result.components}}
+                {{'{{^prevUrl}}'}}
```

Comment:
I wanted to write: "can we avoid the double `{{` this looks tedious to maintain". However, looking at this again, I have to say that I prefer this over `raw` as well, as it's clear now which templates are rendered client-side.

Different tags would be even nicer though...but that would probably be a pain to refactor.

Reply:
I prefer it, too, because of two reasons:
1. The `raw` way had a reversed logic when it comes to the translation strings. So to print out `{{_('My localized string!')}}` it must be `{% endraw %}` right before the string and `{% raw %}` again right after the string, what makes it confusing when reading.
2. When you use a syntax highlighting theme for jinja templates, the whole `raw` section looks like commented out.

The screenshot show both problems:
<img width="969" alt="Screenshot" src="https://user-images.githubusercontent.com/22053023/70794135-338a6400-1d9d-11ea-9b13-bde2c5516405.png">

I don't see any use of refactoring the tags, because they are the standard tags of jinja templating. After understanding that the `<amp-mustache>` variables have to be fully "printed" to the file and don't get rendered, it makes sense that we use ` {{'{{^prevUrl}}'}}` to print out a "string" in jinja.